src/collections/blog/2021-04-16-meet-the-maintainer-jash-patel/index.mdx

```diff
@@ -0,0 +1,225 @@
+    It was so smooth. I was delighted to see the overwhelming response of the community.
+    I loved the way the community encouraged every newcomer to their first contribution.
+    After that, I never looked back. I still remember my first community meeting
+    in which I introduced myself, highlighting my interests in React. The response
+    was Lee giving an overly positive reaction.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Ruth:</span>
+  <p>
+    I can totally relate to how amazing the Layer5 onboarding process is unique!
+    Layer5 Projects has many active, open-source projects. You’ve been
+    consistently contributing to a few of them. Which one(s) are you currently
+    focusing on? Psst. Also, which one’s your favorite? I won’t tell.
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Jash:</span>
+  <p>
+    I started contributing to docs and soon dived into Layer5 websites. While I
+    have worked on almost all websites, t (a little in some). The next
+    generation Layer5 website (built on Gatsby) was the site I worked on the
+    longest. We recently launched this new site (it takes a lot to launch a
+    website). Currently, I am working on enhancements to the website and on
+    Layer5’s upcoming discussion forum. I think, after this, you would already
+    know my favorite one.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Ruth:</span>
+  <p>
+    Definitely, winks! Have you worked with any other open source project? How
+    does Layer5 compare?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Jash:</span>
+  <p>
+    I am a part of my college’s open-source organization. I have contributed to
+    and also have mentored a few open-source projects there. Open source is
+    always fun - it doesn't matter where you do it. The difference I found here
+    was the vast and diverse community that was missing there. Also, exposure to
+    different technologies was advantageous for me.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Ruth:</span>
+  <p>
+    You know for me, coming to The Next Generation Layer5 Website was done in
+    the space of 6 months which is really impressive. How was the team able to
+    pull that off?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Jash:</span>
+  <p>
+    It was a pretty big task. Many people have worked on it(At the time of
+    launch we had 140 contributors). It started with Tanuj and me, and then got
+    many hands to help us and strive towards its completion. I can’t write all
+    those names, or else it would take an entire page to note them down. And
+    this is what communities are meant for; open source is intended for. It was
+    a combined effort, and we worked on almost everything. The team worked on
+    Gatsby, GraphQL, SEO, Analytics, and anything related to the website. The
+    two mentors Lee and Nikhil, were quite helpful for the team to bring it
```

Comment:
@blossom-babs Great work overall. Thank you for this.
Can you add profile links to Lee and Nikhil as well?

Reply:
Okay, I'll do that